Makefile

```diff
@@ -87,6 +89,20 @@ fuzz-linux : export S2N_UNSAFE_FUZZING_MODE = 1
 fuzz-linux : bin
 	$(MAKE) -C tests fuzz
 
+.PHONY : coverage
+coverage: 
+	lcov -c -i -d ${S2N_ROOT} -o ${COVERAGE_DIR}/all_coverage.info
+	$(MAKE) -C bin gcov lcov
```

Comment:
Is there a better way to do this for each directory automatically? Then if we add a new one (such as pq-crypto/) we get coverage data automatically and without forgetting to turn it on.

Reply:
`gcov` needs to be run in the same directory that the code was compiled in. We could probably write a bash script to replicate what this Makefile is doing, but I think that bash script would be more complex than this Makefile. When we add the `pqcrypto` directory we'll need to add another line or two to the Makefiles to support coverage metrics which doesn't seem that bad.